.github/actions/gradle/experiment-1/action.yml

```diff
@@ -103,8 +103,8 @@ runs:
         
         echo "buildScanFirstBuild=$BUILD_SCAN_1" >> $GITHUB_OUTPUT
         echo "buildScanSecondBuild=$BUILD_SCAN_2" >> $GITHUB_OUTPUT
```

Comment:
I would continue to list the receipt files under the `Artifacts` section: (in case someone wants to download them)

<img width="1366" alt="image" src="https://user-images.githubusercontent.com/231070/204192967-9dc376a3-7c2f-47ac-a66c-c9ea5de09f8f.png">

Reply:
It seems this is still the case. Please resolve the comment if it is obsolete.

Reply:
Yes I kept the receipts as workflow artifacts